package io.github.analuizaat.exception;

public class PedidoNaoEncontradoException extends RuntimeException {

    public PedidoNaoEncontradoException() {
        super("Pedido n√£o encontrado.");
    }
}